src/test/java/org/jenkinsci/plugins/ewm/casc/ConfigAsCodeTest.java

```diff
@@ -108,7 +109,7 @@ public void shouldSupportConfigurationAsCodeAgentProperty() throws Exception {
         List<NodeDiskPool> nodeDiskPools = node.getNodeProperties().get(ExternalWorkspaceProperty.class).getNodeDiskPools();
     }
 
-    @Test
+    @ConfiguredWithCode("configuration-as-code.yml")
     public void exportConfiguration() throws Exception {
```

Comment:
This test needs to be completely rewritten

Reply:
I will rewrite this.